addons/im_livechat/demo/im_livechat_channel/im_livechat_session_1.xml

```diff
@@ -95,5 +95,13 @@
         </record>
         <function model="discuss.channel" name="rating_apply"
             eval="([ref('im_livechat.livechat_channel_session_1')], 5, 'LIVECHAT_1', None, 'Good Job')"/>
+        <record id="livechat_channel_session_1_call_1" model="discuss.call.history">
```

Comment:
It should be linked to a notification message in order for the demo data to be realistic.

see `<div data-oe-type="call" class="o_mail_notification"></div>`.